src/components/flowchart/index.js

```diff
@@ -139,9 +147,12 @@ export class FlowChart extends Component {
    * Render chart to the DOM with D3
    */
   drawChart() {
-    const { layout, textLabels } = this.props;
+    const { focusedNode, layout, linkedNodes, textLabels } = this.props;
     const { nodes, edges } = layout;
 
+    // Only apply callback if focusedNode is truthy
+    const ifFocused = callback => (focusedNode ? callback : false);
```

Comment:
Would "isFocused" be a better name?

Reply:
I'd usually use `isFocused` for a Boolean flag, whereas this is a function that executes a callback only if focusedNode is truthy. So it makes more sense to me to call it `ifFocused`, because it will only execute the function if focused (e.g. `.classed('node--highlight', ifFocused(node => linkedNodes[node.id]))`)

Reply:
Ah ok makes sense 👍